docs/en/UsersGuide.md

```diff
@@ -0,0 +1,140 @@
+It is currently distributed for Google Chrome at [Chrome Web Store](https://chromewebstore.google.com/detail/accessibility-visualizer/idcacekakoknnpbfjcdhnkffgfbddnhk).
+
+## What you can do with Accessibility Visualizer
+
+- Image alt text, heading levels, form labeling, and other situations can be displayed overlays on web pages
+- A warning message will be displayed for any of these that are clearly problematic or should be used with caution.
+- `role="status"` `role="alert"` `role="log"` `aria-live` attribute, to notify visually of changes in the live region created by the `<output>` element To do
+
+Until now, this information has only been available through reading the accessibility tree (Accessibility Object Model) in the browser's developer tools, reading the source code, or actually interacting with it using a screen reader. There was no.
+
+Knowledge is required to decipher the accessibility tree and the source code. Screen readers are unfamiliar to most people, and their unique controls can be confusing.
+The goal of Accessibility Visualizer is to visualize this information to eliminate these barriers and enable everyone to be aware of accessibility when coding or checking operation.
+
+Note: Accessibility Visualizer does not completely eliminate the need for actual assistive technology verification, such as screen readers. The information Accessibility Visualizer displays is supplementary and may cause you to miss important issues. **We recommend using screen reader confirmation** to incorporate it into your web development workflow.
+
+## How to use Accessibilty Visualizer
+
+After installing Accessibility Visualizer, an Accessibility Visualizer item will appear in the extension menu.
+If you use it often, we recommend pinning it to your browser's toolbar. If you pin it to the toolbar, the Accessibility Visualizer icon will always be displayed.
+
+![Screenshot of extensions menu open](./images/extensions_menu.png)
+
+When you click the Accessibility Visualizer item in the extension menu or the Accessibility Visualizer icon pinned to the toolbar, the Accessibility Visualizer pop-up opens.
+
+![Screenshot of Accessibility Visualizer popup opened](./images/a11y_visualizer_popup.png)
+
+### Tip display
+
+If you check the "Show tip" checkbox in the popup, various information will be displayed as a "tip" on the web page you are viewing.
+
+!["Komaru City" logo and the heading "Global Warming Prevention Division." The logo is surrounded by a green dotted line and covered by a red chip that says ``Image without alt attribute.'' The logo and headline are surrounded by a blue dotted line, and above them are a blue tip for ``Heading level 1'' and a green tip for ``Global Warming Prevention Division''](./images/tip_example_komaru_city.png )
+
+(Screenshots are from [Komaru City - Experience site for accessibility issues -] (https://a11yc.com/city-komaru/). /city-komaru/practice/?preset=ng-terrible1&wcagver=22)

```

Comment:
The link is broken, and the original web site doesn't seem to have English version, It should written in original language.

```suggestion
(Screenshots is from [たいへんな駒瑠市](https://a11yc.com/city-komaru/practice/?preset=ng-terrible1&wcagver=22) of [駒瑠市〜アクセシビリティ上の問題の体験サイト〜](https://a11yc.com/city-komaru/). /city-komaru/practice/?preset=ng-terrible1&wcagver=22)
```